### PR TITLE
fix: parsing params in available connections api request

### PR DESCRIPTION
**Pull request by Abhikumar98**
# FIX

Parsing all params to get the available connections [LF-3826]

## TL;DR

Fix the issue where only selected params with array value were being parsed, leaving out other array-based params. Now all array params and all values are parsed and passed in the API request.

## DESCRIPTION

Previously, there was an issue where only selected parameters with array values were being parsed, resulting in the exclusion of other array-based parameters. This fix addresses that issue by ensuring that all array parameters and their respective values are parsed and correctly passed in the API request.

## TESTING

No testing is required as this fix is already integrated and tested.

## CHECKLIST

- [x] I have performed a self-review of my code.
- [ ] I have commented the code I wrote to ensure clarity.
- [x] I have made sure that existing tests related to this fix are passing.
- [x] I have verified that the fix resolves the issue described in the ticket.

[LF-3826]: https://lifi.atlassian.net/browse/LF-3826?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/services/ApiService.unit.spec.ts [hunk 1]
```diff
@@ -704,11 +704,16 @@ describe('ApiService', () => {
         fromToken: findDefaultToken(CoinKey.USDC, ChainId.BSC).address,
         toToken: findDefaultToken(CoinKey.USDC, ChainId.OPT).address,
         allowBridges: ['connext', 'uniswap', 'polygon'],
+        allowExchanges: ['a', 'b', 'c'],
+        denyBridges: ['testValue1', 'testValue2'],
+        preferBridges: ['testValue1', 'testValue2'],
+        denyExchanges: ['testValue1', 'testValue2'],
+        preferExchanges: ['testValue1', 'testValue2'],
```
Comment (chybisov) [inline src/services/ApiService.unit.spec.ts:711]: Should we add real data to this test?

**Review by chybisov — CHANGES_REQUESTED** [review]

**Review by chybisov — APPROVED** [review]
